frontend/src/pages/GraphPF/GraphPagePF.tsx

```diff
@@ -85,7 +86,24 @@ export type GraphURLPathProps = {
   workload: string;
 };
 
-type ReduxProps = {
+type DispatchProps = {
```

Comment:
It is due to return type eslint rule. Since we have two redux methods: `mapStateToProps` and `mapDispatchProps`, we need two return types: `ReduxStateProps` and `ReduxDispatchProps` . @josunect I see that you have called them `StateProps` and `DispatchProps`. IMO they should have `Redux` prefix to know that they are part of Redux properties.

Reply:
This is because we need to specify the return type for the lint, and that specific type is used here: https://github.com/kiali/kiali/pull/7015/files/3d4da6819f7c6d757123206b0df14089cad9a08f#diff-e716bd7ad252c199194834133006e711cade52b449bfa99d3593c60f6b86be2aR767 and here: https://github.com/kiali/kiali/pull/7015/files/3d4da6819f7c6d757123206b0df14089cad9a08f#diff-e716bd7ad252c199194834133006e711cade52b449bfa99d3593c60f6b86be2aR728

Reply:
> It is due to return type eslint rule. Since we have two redux methods: `mapStateToProps` and `mapDispatchProps`, we need two return types: `ReduxStateProps` and `ReduxDispatchProps` . @josunect I see that you have called them `StateProps` and `DispatchProps`. IMO they should have `Redux` prefix to know that they are part of Redux properties.

Ops I haven't seen this!

Reply:
Thanks for the explanation!  I agree with @ferhoyos, please prefix it so that the names are ReduxDispatchProps and ReduxStateProps.